### PR TITLE
added create-bcktlist and view-bcktlist page

### PR DESCRIPTION
**Pull request by iannech**
I have created a for publishing a new bucket list and a page to display 'my' bucket lists. Is the design good enough?

### CODE REVIEW

File: templates/addWish.html [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+</head>
+ 
+<body>
+ 
+    <div class="container">
+        <div class="header">
+            <nav>
+                <ul class="nav nav-pills pull-right">
+                    <li role="presentation" class="active"><a href="#">Create New</a></li>
+                    <li role="presentation"><a href="/userHome">My Bucket Lists</a>
```
Comment (khwilo) [inline templates/addWish.html:24]: Missing closing tag
```html
</li>
```

File: templates/signup.html [hunk 2]
```diff
@@ -17,12 +17,12 @@
       <div class="header">
         <nav>
           <ul class="nav nav-pills pull-right">
-            <li role="presentation" ><a href="main">Home</a></li>
+            <li role="presentation" ><a href="/">Home</a></li>
             <li role="presentation"><a href="showSignIn">Login</a></li>
             <li role="presentation" class="active"><a href="#">Register</a></li>
```
Comment (khwilo) [inline templates/signup.html:22]: Why use a stub page here? Is the user allowed to sign in only?

**Review by khwilo — CHANGES_REQUESTED** [review]
Clean code with minor changes to be done.

**Comment by iannech** [comment]
Noted, the changes will be made ASAP. thanks

**Pull request merged**